src/components/dropdown/dropdown.js

```diff
@@ -649,6 +652,15 @@ class Dropdown extends React.Component {
     return results;
   }
 
+  positionOptions() {
+    if (this.dropDown) {
+      const input = this.dropDown.getElementsByClassName('carbon-dropdown__input common-input__input')[0];
```

Comment:
I think all input components have a ref available of `input` or `_input` or something - you should just be able to use this.

Reply:
looks like `_input` - https://github.com/Sage/carbon/blob/master/src/utils/decorators/input/input.js#L236

Reply:
done